plugins/foreman_openscap/0.4/index.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: plugin
+title: Foreman OpenSCAP Manual
+images: /plugins/foreman_openscap/0.4
+version: 0.4.1
+---
+
+# 1. {{ page.title }}
+
+This plugin enables Foreman to receive automated vulnerability assessment and security complinace audits from Foreman hosts.
+You can upload SCAP compliance contents, create compliance policies out of them and assign the policies to hosts or hostgroups.
+foreman_openscap plugin provides three default SCAP contents, so you could start testing security compliance on RHEL6/7 and Fedora.
+
+OpenSCAP reports (aka arf reports) will help you find vulnerabilities on your hosts and also suggest remediation plan to fix those vulnerabilities.
+![OpenScap report]({{page.images}}/report_info_details.png)
+
+Foreman OpenSCAP plugin is made of 5 components:
+
+* Scaptimony - Rails engine which creates and persists SCAP content, Compliance policy and ARF report objects
+* foreman_openscap - UI to display Scaptimony engine (which actually connects to OpenSCAP)
+* smart_proxy_openscap - Smart-Proxy plugin which distributes SCAP content to hosts and post ARF reports from client hosts to Foreman
+* foreman_scap_client - A client script which runs OpenSCAP scan and uploads the scan report to the Smart-Proxy
+* puppet-foreman_scap_client - A puppet module which configures foreman_scap_client
+
+## 1.1 OpenSCAP Basic Concepts
+
+There are three basic concepts (entities) in OpenSCAP plug-in: SCAP Contents, Compliance Policies and ARF Reports.
+
+__SCAP Content__ represents SCAP DataStream XML file as defined by SCAP 1.2 standard. Datastream file contains implementation of compliance, configuration or security baselines. Users are advised to acquire examplary baseline by installing scap-security-guide package. DataStream file usualy contains multiple XCCDF Profiles. Each for different security target. The content of Datastream file can be inspected by oscap tool from openscap-scanner package.
```

Comment:
A quick Google search reveals not even open-scap.org is consistent using one or the other :smile: